src/components/users/Accounts.jsx

```diff
@@ -43,12 +50,18 @@ export function getAccountsState (
     userName = "",
     password = "",
     confirmPassword = "",
+    rootPassword = "",
+    rootConfirmPassword = "",
+    enableRoot = false,
```

Comment:
Maybe I should rather use rootEnabled or isRootEnabled here.

Reply:
Self review is the best review :P

Reply:
I like isRootEnabled :)